Meta/Lagom/CMakeLists.txt

```
@@ -101,6 +101,8 @@ if (EMSCRIPTEN)
     add_link_options(--emrun "SHELL:-s ALLOW_MEMORY_GROWTH")
 endif()
 
+set(LINKER_FLAGS "${LINKER_FLAGS} -lpthread")
```

```suggestion
find_package(Threads REQUIRED)
link_libraries(Threads::Threads)
```

or perhaps just `target_link_libraries(LibCore PUBLIC Threads::Threads)` after the find_package

.. but we already have that. Did we delete the find_package accidentally or something?

---

Ah yeah, just adding `link_libraries(Threads::Threads)` does the trick. Though I'm confused that it seems I'm the only one encountering this issue :thinking:

---

I'm ok adding the global link libraries, but could you create an issue and link it with a FIXME comment just above it? 

I don't remember seeing that behavior you mentioned a few days ago when testing out the serenity.sh script, so it probably warrants a closer look.